recipe/build-mpi.sh

```diff
@@ -33,15 +36,19 @@ if [[ "$target_platform" == osx-* ]]; then
     wrapper_ldflags='-Wl,-rpath,${libdir}'
 fi
 
-# UCX support
+# UCX/UCC support
 build_with_ucx=""
+build_with_ucc=""
 if [[ "$target_platform" == linux-* ]]; then
```

Comment:
I think this is where you need a check for `${target_platform} != "linux-ppc64le"`

Reply:
It works!